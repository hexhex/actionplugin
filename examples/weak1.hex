a v b.
b v c.
d v -d :- a,c.
:~ b. [1:2]
:~ a, -d. [4:1]
:~ c, d. [3:1]
g :- b.
:~g. [1:1]
