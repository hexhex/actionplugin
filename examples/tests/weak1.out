{d, c, a}
Cost ([Weight:Level]): <[3:1],[0:2]>
{-d, c, a}
Cost ([Weight:Level]): <[4:1],[0:2]>
{g, b}
Cost ([Weight:Level]): <[1:1],[1:2]>
